docs/design/proposals/CG-Common-Design-Goals.md

```diff
@@ -0,0 +1,92 @@
+# Design notes for grouping volumes for consistency group based DR
+
+## Grouping decision
+
+DRPC will carry an additional `spec.group` (or, `spec.async.group`) boolean,
```

Comment:
We aimed for an annotation in this PR: https://github.com/RamenDR/ramen/pull/1488